### PR TITLE
Marvin commands

### PR DESCRIPTION
**Pull request by thokkat**
This PR implements the following marvin commands:
```
cheat god
kill
aigoto 
goto waypoint
goto pos 
```

### CODE REVIEW

File: game/game/playercontrol.cpp [hunk 1]
```diff
@@ -430,6 +430,7 @@ void PlayerControl::marvinF8(uint64_t dt) {
   pl.clearSpeed();
   pl.quitIneraction();
   pl.setAnim(AnimationSolver::Idle);
+  pl.clearAiQueue();
```
Comment (Try) [inline game/game/playercontrol.cpp:433]: Is it intentional change? Commit message implies that it is not

File: game/gothic.h [hunk 2]
```diff
@@ -86,6 +86,9 @@ class Gothic final {
     bool         isMarvinEnabled() const;
     void         setMarvinEnabled(bool m);
 
+    bool         isGodMode() const {return godMode; }
+    void         setGodMode() { godMode = !godMode; }
```
Comment (Try) [inline game/gothic.h:90]: `set` function take no arguments, therefore it's more of `toogleGodMode`. 
Can you please change this to `setGodMode(bool)`?

File: game/marvin.cpp [hunk 3]
```diff
@@ -237,6 +243,67 @@ bool Marvin::exec(std::string_view v) {
...
+      player->aiPush(AiQueue::aiGoToPoint(*wpoint));
+      return true;
+      }
+    case C_GoToPos: {
+      Npc* player = Gothic::inst().player();
+      if(player==nullptr || !player->setInteraction(nullptr))
+        return false;
+      float x,y,z;
+      try {
+        x = std::stof(ret.argv[0].data());
```
Comment (Try) [inline game/marvin.cpp:280]: Usage of `stof` with `string_view` is not correct, since string_view is not zero-terminated.
Need to use `std::from_chars`

**Review by Try — CHANGES_REQUESTED** [review]

File: game/game/playercontrol.cpp (hunk 1 above)
Comment (thokkat) [reply game/game/playercontrol.cpp:433]: F8 stops `aigoto`  and all further queued `aigoto` commands are ignored.

**Comment by thokkat** [comment]
Did the requested changes + some cosmetics.

**Comment by Try** [comment]
Merged, thanks!

**Pull request merged**